operate/client/src/modules/hooks/modifications.ts

```diff
@@ -19,6 +20,27 @@ import {
 import {processInstanceDetailsDiagramStore} from 'modules/stores/processInstanceDetailsDiagram';
 import {useMemo} from 'react';
 
+const useWillAllFlowNodesBeCanceled = () => {
+  const {data: statistics} = useFlownodeInstancesStatistics();
+
+  if (
+    modificationsStore.flowNodeModifications.filter(({operation}) =>
+      ['ADD_TOKEN', 'MOVE_TOKEN'].includes(operation),
```

Comment:
Don't we have constants for `'ADD_TOKEN'` and `'MOVE_TOKEN'`? It's a bit dangerous to have these floating strings like these

Please check if we don't have constants for this already, if we don't, please create it.